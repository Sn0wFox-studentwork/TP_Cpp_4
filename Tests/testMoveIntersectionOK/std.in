R r1 2 2 0 0
R r2 5 2 3 2
OI r12 r1 r2
MOVE r12 3 3
LIST
EXIT