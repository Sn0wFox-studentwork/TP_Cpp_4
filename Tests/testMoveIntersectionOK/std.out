OK
OK
OK
OK
R r1 2 2 0 0
OI r12 {
R 5 5 3 3
R 8 5 6 5
}
R r2 5 2 3 2